test/utils.toTwosComplement.js

```diff
@@ -7,7 +7,7 @@ var utils = require('../packages/web3-utils/');
 var BigNumber = require('bignumber.js');
 
 var tests = [
-    { value: 1, expected: utils.padLeft(new BigNumber(1).round().toString(16), 64) },
+    { value: 1, expected: utils.padLeft(new BigNumber(1).integerValue().toString(16), 64) },
```

Comment:
Are we introducing some potential breaking features by bumpng to `bignumber.js: 9.0.0` ?

Reply:
ah, i see it was being used previously at two different versions

Reply:
yes this was the only issue I encountered bumping up the version, I’ll keep an eye on any other side effects